llvm/test/TableGen/PreWriteCycleCount.td

```diff
@@ -0,0 +1,41 @@
+// RUN: llvm-tblgen -gen-subtarget -I %p/../../include %s 2>&1 | FileCheck %s
```

Comment:
File name has a typo "PreWrite" instead of "PerWrite"?

Reply:
You're right of course, thanks for catching this.